script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -219,26 +219,27 @@ But he doesn't smell like a bourgeois at all, does he?!"
 [sha:d22831132d02817cc2bdb11c371bc4670e7152e4]{
 -- Page 107, Offset 19937.
 -- 「どんな匂いだソレ。金に匂いなんてねえだろ。それとも手垢の匂いってコトか？　金持ちはドイツもコイツも垢だらけってか？　ずいぶん舐めた発言だなオイ？」
-"What kind of smell is that? It's not like you can smell money, right?
- Or are you talking about the smell of the filth on his hands?
+// Since Andou says he didn't smell anything, it'd be weird for Mario to insinuate he did smell something on Shiki's hands. He's talking more generally
+"And what kinda smell would that be? Think you can smell money?
+ Or are you talking about the filthy smell of their hands?
  Are you saying that's all the wealthy are? Filth?
- Those sound like fighting words to me, oi".
+ Those sound like fighting words to me, fucker.".
 }
 [sha:458fec36165274bdde497731b1c7b8981dc1e84f]{
 -- Page 108, Offset 19938.
 -- 「い、いえ、違います、成金だけ、一部の成金だけですってば！　ほら、一日の半分を金勘定で浪費する高利貸し、みたいな？」
-"N-No, not at all, I just mean the %{i}nouveau riche%{/i}, and only a handful of them at that!
+"N-No, not at all. Maybe some of the %{i}nouveau riche%{/i} do, but that's all!
```

Comment:
I think the continuation here is "Maybe some of the nouveau riche [are filth]", since that's the last statement by Mario, so s/do/are if so